app/src/main/res/drawable/ic_tracking_protection_disabled.xml

```diff
@@ -7,21 +7,10 @@
     android:height="24dp"
     android:viewportWidth="24"
     android:viewportHeight="24">
-    <group>
-        <clip-path android:pathData="M 18.8 2.8 C 19.8 1.8 21.1 3.3 20.2 4.2 L 3.4 21 L 2 19.6 L 18.8 2.8 Z" />
-        <path
-            android:name="strike_thru_path"
-            android:fillColor="@color/disabled_text"
-            android:pathData="M 20 1.6 L 21.4 3 L 5.2 19.2 C 4.2 20.2 2.8 18.8 3.8 17.8 L 20 1.6 Z"
-            android:strokeWidth="1" />
-    </group>
-    <group>
-        <clip-path
-            android:name="strike_thru_mask"
-            android:pathData="M 0 0 L 0 24 L 24 24 L 24 0 L 0 0 Z M 21 2 L 23 4 L 6 21 L 4 19 L 21 2 Z" />
-        <path
-            android:name="icon"
-            android:fillColor="@color/disabled_text"
-            android:pathData="M20 6c0-1-0.8-1.9-1.8-2L12 3 5.8 4C4.8 4 4 5 4 6l0.1 5c0.3 3.2 1 5 2.5 7a8.4 8.4 0 0 0 5.3 3h0.2c2.1-0.3 4-1.4 5.3-3 1.6-2 2.2-3.8 2.5-7l0.1-5zm-2.1 4.8a10 10 0 0 1-2 6c-1 1.1-2.4 2-3.9 2.3a6.5 6.5 0 0 1-3.9-2.4 9.9 9.9 0 0 1-2-5.9 67.3 67.3 0 0 1 0-4.9L12 5l5.9 1 0.1 0.2-0.1 4.7zM8 7.6v3c0.3 2.7 0.8 3.7 1.7 5 0.6 0.6 1.4 1.2 2.3 1.4V7l-4 0.6z" />
-    </group>
+    <path
+        android:pathData="M3.53,3.22C3.237,2.927 2.762,2.927 2.469,3.22C2.176,3.513 2.176,3.988 2.469,4.281L4.069,5.881C3.158,6.571 2.698,7.719 2.909,8.872L3.617,12.717C3.888,14.198 4.386,15.493 5.095,16.567C6.525,18.736 8.496,20.547 10.647,21.669C11.066,21.885 11.534,22 12,22C12.468,22 12.937,21.885 13.355,21.667C14.687,20.972 15.944,20.007 17.051,18.862L19.47,21.281C19.616,21.427 19.808,21.501 20,21.501C20.192,21.501 20.384,21.428 20.53,21.281C20.823,20.988 20.823,20.513 20.53,20.22L3.53,3.22ZM12.663,20.337H11.339C9.412,19.332 7.639,17.7 6.347,15.74C5.75,14.836 5.327,13.727 5.092,12.445L4.338,8.368L4.844,7.131L5.154,6.965L15.993,17.804C14.995,18.841 13.861,19.711 12.663,20.337Z"
+        android:fillColor="@color/disabled_text"/>
```

Comment:
We should perhaps create a separate variable for this? Seems weird to have the text color be unexpectedly used for icon fill colors. Can also file this as a followup

Reply:
Sure https://github.com/mozilla-mobile/android-components/issues/10165